_episodes_rmd/01-rstudio-intro.Rmd

```diff
@@ -92,7 +92,7 @@ in the top left.
 There are two main ways one can work within RStudio:
 
 1. Test and play within the interactive R console then copy code into
-a .R file to run later.
+a R script(.R file) to run later.
```

Comment:
I think this should be "an R script (.R file)"

Note:
a -> an 
space before the parentheses